### PR TITLE
Capstone Project 1: Conferences Page

### PR DESCRIPTION
**Pull request by taldr27**
## Conferences Page
In this Capstone project I: 

- Added index.html and about.html for each page.
- Added variables in the root class inside of the CSS file.
- Added the correct function to generate the speakers' section on the HTML file.
- Added some images related to the topic that I choose.
I leave the Loom video link here: [Loom Video](https://www.loom.com/share/609c9b4807d04d159eacd25a5bc74da7)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,188 @@
...
+                            <li><a href="about.html">About</a></li>
+                            <li><a href="#">Program</a></li>
+                            <li><a href="#">Join</a></li>
+                            <li><a href="#">Sponsor</a></li>
+                            <li><a href="#">News</a></li>
+                            <li class="special-li"><a href="#">CC Campaign</a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
```
Comment (leonardodiasb) [inline index.html]: - [x] These two bars are the navbar and they would be better grouped in the same `<nav>` tag together, and maybe they should not be part of this header section. There is no need to have two navbars. Please group the navbars together in the same `<nav>` tag.

File: index.html [hunk 2]
```diff
@@ -4,9 +4,188 @@
...
+                            <p>Try creating your own music with your favorite artists and bands</p>
+                        </div>
+                        <div class="program-categories-details">
+                            <img src="images/program_icon_05.png" alt="">
+                            <span>GGC Ignite</span>
+                            <p>Get opportunities to network with GGC affiliates around the world, also after the summit.</p>
+                        </div>
+                        
+                    </div>
+                    <a class="btn-mobile-program">SEE THE WHOLE PROGRAM</a>
```
Comment (leonardodiasb) [inline index.html:134]: - [x] Your ''SEE THE WHOLE PROGRAM' link has the same color as the background and does not appear in the desktop version. Please use the colors from the color swatch to make it appear
 -Your design:
image

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,736 @@
...
+    min-width: var(--size-small);
+    max-height: 35em;
+    gap: 0.3em;
+    justify-items: center;
+  }
+
+  .program-categories-details {
+    flex-direction: column;
+    min-width: 10%;
+  }
```
Comment (leonardodiasb) [inline styles/style.css]: - [x] Your program cards have inconsistent width on the desktop version. Some are larger than others making it look not professional. Please make sure they have a consistent design like the template you followed.
 - [x] On the desktop version of the template, the text alignment is centered. Kindly, fix the alignment to match the template design.
 -Your design:
image
-Template design:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @taldr27, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Very professional commit history. ✔️
 - Great visual design and very responsive. ✔️ 
 - Nice video presentation. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @taldr27 
### I'm Levy, your PR reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**